toucan_connectors/common.py

```diff
@@ -154,18 +151,11 @@ def _handle_missing_params(elt: dict | list[dict] | tuple | str, params: dict, h
         for k, v in elt.items():
             if isinstance(v, str):
                 missing_params = []
-                # In case the query was interpolated before being passed to the connector
-                if v.strip() in _PARAMETER_VALUES_TO_ELIMINATE:
-                    continue
                 matches = re.findall(RE_PARAM, v) + re.findall(RE_JINJA, v)
 
                 for m in matches:
                     try:
-                        rendered = Template('{{ %s }}' % m, undefined=StrictUndefined).render(
-                            params
-                        )
-                        if rendered in _PARAMETER_VALUES_TO_ELIMINATE:
-                            missing_params.append(m)
+                        _ = Template('{{ %s }}' % m, undefined=StrictUndefined).render(params)
```

Comment:
Is there a point in keeping this function ?

Reply:
yes, that Template match is for mongo "Undefined" values not related to "__VOID__"

Reply:
And don't you think that should be handled by `_render_query` ?